test/PoolManager.t.sol

```diff
@@ -325,11 +332,18 @@ contract PoolManagerTest is Test, Deployers, GasSnapshot {
         assertEq(manager.balanceOf(address(this), currency0.toId()), 0);
         assertEq(manager.balanceOf(address(this), currency1.toId()), 0);
 
+        uint256 currency0BalanceBefore = currency0.balanceOfSelf();
+        uint256 currency1BalanceBefore = currency1.balanceOfSelf();
+
         // remove liquidity as 6909: settleUsingTransfer=false (unused), withdrawTokens=false
         modifyLiquidityRouter.modifyLiquidity(key, REMOVE_LIQ_PARAMS, ZERO_BYTES, false, false);
 
         assertTrue(manager.balanceOf(address(this), currency0.toId()) > 0);
         assertTrue(manager.balanceOf(address(this), currency1.toId()) > 0);
+
+        // ERC20s are unspent
+        assertEq(currency0.balanceOfSelf(), currency0BalanceBefore);
```

Comment:
Ah yes this check is good to add cause yeah our ERC20 balance shouldn't change but I also think the manager's balance shouldn't change yeah? so doing like 

currency0.balanceOf(address(manager)) is a nice thing to add